### PR TITLE
feat(ng-dev/release): properly mark releases as latest on GitHub 

### PR DESCRIPTION
**Pull request by devversion**
See individual commits

### CODE REVIEW

File: ng-dev/release/publish/actions.ts [hunk 1]
```diff
@@ -635,6 +638,7 @@ export abstract class ReleaseAction {
       name: `v${releaseNotes.version}`,
       tag_name: tagName,
       prerelease: isPrerelease,
+      make_latest: shouldShowAsLatestOnGitHub ? 'true' : 'false',
```
Comment (josephperrott) [inline ng-dev/release/publish/actions.ts:641]: Woohoo! I didn't know they finally completed my feature request!

File: ng-dev/release/publish/actions.ts [hunk 2]
```diff
@@ -605,11 +605,14 @@ export abstract class ReleaseAction {
...
    *   will point to this commit.
    * @param isPrerelease Whether the new version is published as a pre-release.
+   * @param shouldShowAsLatestOnGitHub Whether the version released will represent
+   *   the "latest" version of the project. I.e. GitHub will show this version as "latest".
    */
   private async _createGithubReleaseForVersion(
     releaseNotes: ReleaseNotes,
     versionBumpCommitSha: string,
     isPrerelease: boolean,
+    shouldShowAsLatestOnGitHub: boolean,
```
Comment (josephperrott) [inline ng-dev/release/publish/actions.ts:615]:
```suggestion
    useAsLatest: boolean,
```

**Review by josephperrott — APPROVED** [review]
LGTM

File: ng-dev/release/publish/actions.ts (hunk 2 above)
Comment (devversion) [reply ng-dev/release/publish/actions.ts:615]: I had that initially but went with this to make it very clear what it's doing. "latest" itself in this context is quite an ambiguous term.

Comment (devversion) [reply ng-dev/release/publish/actions.ts]: I've renamed it to omit the `should`, but other than that- I think this term is quite good

**Pull request merged**